test/cpp/naming/resolver_component_tests_runner.py

```diff
@@ -606,7 +637,8 @@ def _quit_on_signal(signum, _frame):
   '--enable_srv_queries', 'True',
   '--enable_txt_queries', 'True',
   '--inject_broken_nameserver_list', 'False',
-  '--local_dns_server_address', '127.0.0.1:%d' % args.dns_server_port])
+  '--local_dns_server_address', '127.0.0.1:%d' % args.dns_server_port
+  ] + filter(lambda x: x, args.extra_args.split(',')))
```

Comment:
optional: The 32 test cases could be auto-generated to improve readability.

Reply:
They are already auto-generated -- `test/cpp/naming/resolver_component_tests_runner.py` is generated from `templates/test/cpp/naming/resolver_component_tests_defs.include`

Reply:
Good to learn that